Dynamic swap fee rate controled by pool governor (#249)
tendermint/liquidity: x/liquidity/spec/04_messages.md

  - if `OrderPrice` <= zero
  - if `OfferCoinFee` Equal `OfferCoin` _ `params.SwapFeeRate` _ `0.5` with truncating Int
  - if has sufficient balance `OfferCoinFee` to reserve offer coin fee.
## MsgSetPoolSwapFeeRate

// Sets new swap fee rate on the pool.

```go
type MsgSetPoolSwapFeeRate struct {

>   PoolId        uint64  // id of the liquidity pool where this message is belong to

Reviewer: ```suggestion
  PoolId        uint64  // id of the liquidity pool that this message belongs to
```

  SetterAddress string 	// swap fee rate setter address. Must be the address of pool governor.
  SwapFeeRate   sdk.Dec // new swap fee rate for the pool
}
```

**Validity check**

> - `MsgSetPoolSwapFeeRate` failes if
>   - `SwapRequester` address does not exist
>   - `PoolId` does not exist
>   - `SwapFeeRate` excedes maximum allowed rate
>   - `SetterAddress` does not belong to the pool governor

Reviewer: ```suggestion
- `MsgSetPoolSwapFeeRate` fails if
  - `SwapRequester` address does not exist
  - `PoolId` does not exist
  - `SwapFeeRate` exceeds maximum allowed rate
  - `SetterAddress` does not belong to the pool governor
```
